src/main/java/jewellery/inventory/model/User.java

```diff
@@ -0,0 +1,66 @@
+package jewellery.inventory.model;
+
+import jewellery.inventory.model.resources.ResourceInUser;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue
+    private UUID id;
+
+    private String name;
+    private String email;
+
+    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL.ALL)
```

Comment:
You have a bit of a typo in CascadeType.ALL.ALL. Second ALL is not needed. I am a bit surprised the IDE didn't complain about it. 
To be fair, I would remove the cascade type for User's fields for now because with CascadeType.ALL when we delete a user, it's going to delete all the products associated with the user and we don't want to do that.